### DISCUSSION TITLE
Change Database Directory.

### DISCUSSION THREAD

**Discussion by haseeb4091** [drkNsubuga/PharmaSpot#18] [Ideas]
Currently Data is storing in App Data inside the C Directory. No backup if windows got corrupted. If we use other than C drive than we have backup in case of windows problem.

**Comment by drkNsubuga** [comment]
Yes, it is. Please provide more details if there is a problem with it.

**Comment by haseeb4091** [comment]
Thanks for your response.
If any pharmacy is using this software.
And there windows got currpted then there data will loss.To prevent this
issue we need to store data in other drive.

**Reply by drkNsubuga** [reply]
This is a design decision for now. However, we'll improve it in future releases by allowing the user to choose the storage folder.

**Reply by haseeb4091** [reply]
Nice.. Looking forward to it.
